dbt/seeds/etl_fast_row_counts.csv

```diff
@@ -596,6 +596,7 @@ core_eia__yearly_fuel_receipts_costs_aggs,2023,32659
 core_eia__yearly_fuel_receipts_costs_aggs,2024,32412
 core_eia__yearly_fuel_receipts_costs_aggs,2025,3809
 core_eiaaeo__yearly_projected_electric_sales,2023,51376
+core_eiaaeo__yearly_projected_energy_use_by_sector_and_type,2023,497073
```

Comment:
This didn't get added when the table was created but running the integration tests locally updated the seeds so I thought I would commit it for completeness.